tests/ctl/core/test_export.py

```diff
@@ -109,6 +109,20 @@ def test_system_records_to_export(
     assert len(output_list) == 3
 
 
+@pytest.mark.unit
+def test_system_records_to_export_sans_privacy_declaration(
+    test_sample_system_taxonomy: Generator, test_config: FidesConfig
+) -> None:
+    """
+    Asserts that unique records for systems without a privacy declaratopm
```

Comment:
```suggestion
    Asserts that unique records for systems without a privacy declaration
```

Reply:
Must have been the fastest I have ever typed 😂